eel_hole/templates/search.html

```diff
@@ -16,9 +16,18 @@
 </div>
 <div id="app" class="columns container is-fluid is-flex-grow-1" x-data="tableState">
   <div class="data-dictionary column my-3 is-flex is-flex-direction-column">
-    <input class="input is-medium block" type="text" name="q" hx-get="/search" hx-trigger="input changed delay:300ms"
-      hx-target="#search-results" hx-replace-url="true" placeholder="Search..." {% if query %}value="{{ query }}" {%
-      endif %} />
+    <input
+      class="input is-medium block"
+      type="text"
+      name="q"
+      hx-get="/search"
+      hx-vals="js:{ferc_enabled:new URLSearchParams(new URL(window.location).search).get('ferc_enabled')}"
```

Comment:
What does `hx-vals` do?

Reply:
Adds URL params to the AJAX request that htmx makes. Here it makes the params dynamically though I wonder if I can just set the param at template-render time.